src/oneD/StFlow.cpp

```diff
@@ -91,6 +102,12 @@ StFlow::StFlow(IdealGasPhase* ph, size_t nsp, size_t points) :
     m_kRadiating.resize(2, npos);
     m_kRadiating[0] = m_thermo->speciesIndex("CO2");
     m_kRadiating[1] = m_thermo->speciesIndex("H2O");
+	
+	// added by UDRI to insure flame control is not applied by default
```

Comment:
This comment is not needed, the commit logs will show the history of the changes, and with useful commit messages, the reasons will be clear as well.